sdk/core/azure-core-tracing-opencensus/README.md

```diff
@@ -80,125 +80,65 @@ A span represents a single operation in a trace. A span could be representative
 ## Examples
 The following sections provides examples of using the azure-core-tracing-opencensus plugin with some of the Azure Java SDK libraries:
 ### Using the plugin package with HTTP client libraries
-- Sync create a secret using [azure-security-keyvault-secrets][azure-security-keyvault-secrets] with tracing enabled.
-    
-    Users can additionally pass the value of the current tracing span to the SDKs using key **"opencensus-span"** on the [Context][context] parameter of the calling method:
-
-    ```java
-    import com.azure.identity.credential.DefaultAzureCredentialBuilder;
-    import com.azure.security.keyvault.secrets.SecretClientBuilder;
-    import com.azure.security.keyvault.secrets.SecretClient;
-    import com.azure.security.keyvault.secrets.models.Secret;
-    import io.opencensus.common.Scope;
-    import io.opencensus.trace.Tracer;
-    import io.opencensus.trace.Tracing;
-    import com.azure.core.util.Context;
-
-    import static com.azure.core.implementation.tracing.Tracer.OPENCENSUS_SPAN_KEY;
-    
-    SecretClient client = new SecretClientBuilder()
-        .endpoint("<your-vault-url>")
-        .credential(new DefaultAzureCredentialBuilder().build())
-        .buildClient();
+Synchronously create a secret using [azure-security-keyvault-secrets][azure_security_keyvault_secrets] with tracing enabled.
+
+Users can additionally pass the value of the current tracing span to the calling method using key PARENT_SPAN_KEY on the [Context][context] parameter.
+
+```java
+    SecretClient client = getSecretClient();
```

Comment:
Is this meant to compile if someone copied and pasted the example?

Reply:
Not entirely sure if we want this one to compile. Especially with this, it then becomes very verbose and the focus quickly shifts to showing client building rather than tracing context passing.